packages/ses/test/test-console-error-trap.js

```diff
@@ -1,17 +1,18 @@
 import test from 'ava';
+import url from 'url';
 import { exec } from 'child_process';
-import { dirname, join } from 'path';
 
-const cwd = join(
-  dirname(new URL(import.meta.url).pathname),
-  'console-error-trap',
-);
+const cwd = url.fileURLToPath(new URL('console-error-trap/', import.meta.url));
```

Comment:
Does this result in an trailing `/` compared to before?

Reply:
This is an effect of moving from path math to URL math. In path math, files and directories are indistinguishable and a final slash would be erroneous, but the domain of URL’s distinguishes directories with the final slash. I can’t imagine the difference being germane in this case, but it didn’t seem to work without the final slash.

Reply:
Right, this is a base, which is used to create more paths later!

Reply:
It seems that the final slash is in fact superfluous here too.

Reply:
Now that confuses me even more. For `..` I understood that going "up" meant it was safe to assume the parent was a folder. However resolving `console-error-trap` could be a file. So if there is no trailing `/` in the relative part, then there wouldn't be a resulting trailing `/` in the absolute path string, unless there is some kind of fs lookup?

Reply:
Ha ok, that's because `cwd` is only ever used by `exec`, not for further path resolution.